src/oscar/apps/dashboard/vouchers/forms.py

```diff
@@ -85,15 +78,30 @@ def clean(self):
 class VoucherSearchForm(forms.Form):
     name = forms.CharField(required=False, label=_("Name"))
     code = forms.CharField(required=False, label=_("Code"))
-    is_active = forms.BooleanField(required=False, label=_("Is Active?"))
-    in_set = forms.BooleanField(
-        required=False, label=_("In Voucherset?"))
+    offer_name = forms.CharField(required=False, label=_("Offer name"))
+    is_active = forms.NullBooleanField(required=False, label=_("Is active?"))
```

Comment:
I find the "Unknown" label generated by this widget a little weird - in the sense that it suggests we're asking to filter on vouchers where we don't know what the status of `is_active` is, rather than that we're asking not to filter on this field at all. Not sure how much this is just me being pedantic though - thoughts?